preprocess/project.go

```diff
@@ -55,6 +91,16 @@ func Preprocess(item interface{}, replaceTypes bool) interface{} {
 	}
 }
 
+func Contains(key string, serviceArray []string) string {
```

Comment:
This function is already available in the `utils` package (https://github.com/rancher/rancher-compose/blob/master/utils/util.go#L44).